kubernetes/redpanda/values.yaml

```diff
@@ -0,0 +1,169 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# values.yaml
+#
+# This file contains values for variables referenced from yaml files in the templates directory.
+#
+# For further information on Helm templating see the documentation at:
+#   https://helm.sh/docs/chart_template_guide/values_files/
+#
+# For an explanation of Redpanda configuration values:
+#   https://vectorized.io/docs/advanced-configuration/
+
+clusterDomain: cluster.local
+
+image:
+  repository: vectorized/redpanda
+  # The imagePullPolicy will default to Always when the tag is 'latest'
+  tag: latest
+
+config:
+  config_file: /etc/redpanda/redpanda.yaml
+  license_key: ""
+  organization: ""
+  redpanda:
+    admin:
+      address: 0.0.0.0
+      port: 9644
+    advertised_kafka_api:
+      address: 0.0.0.0
+      port: 9092
+    advertised_rpc_api:
+      address: 0.0.0.0
+      port: 33145
+    auto_create_topics_enabled: true
+    data_directory: /var/lib/redpanda/data
+    developer_mode: false
+    kafka_api:
+      address: 0.0.0.0
+      port: 9092
+    kafka_api_tls:
+      cert_file: ""
+      enabled: false
+      key_file: ""
+      truststore_file: ""
+    node_id: 0
+    rpc_server:
+      address: 0.0.0.0
+      port: 33145
+  rpk:
+    coredump_dir: /var/lib/redpanda/coredump
+    enable_memory_locking: false
+    enable_usage_stats: true
+    overprovisioned: false
+    smp: 1
+    tls:
+      cert_file: ""
+      key_file: ""
+      truststore_file: ""
+    tune_aio_events: false
+    tune_clocksource: false
+    tune_coredump: false
+    tune_cpu: false
+    tune_disk_irq: false
+  seastar:
+    default_log_level: info
+
+statefulset:
+  replicas: 3
+  # Additional labels to apply to this StatefulSet and all its Pods.
+  labels:
+    app.kubernetes.io/component: redpanda
+  # Additional annotations to apply to the Pods of this StatefulSet.
+  annotations: {}
+
+imagePullSecrets: []
```

Comment:
I don't think this is used anywhere, and is closely related to `image` config, if we keep it, it should be moved closer.

Reply:
I took it out. It is only used when it is anticipated that the Docker image will be in a private repository.

Reply:
In your comment below, which docs are you referring to?

Reply:
https://github.com/vectorizedio/deployment-automation/pull/14/files#r533346889